src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -207,12 +207,12 @@ namespace OpenMS
                 indices.end(),
                 [](const SiriusWorkspaceIndex& i, const SiriusWorkspaceIndex& j) { return i.scan_index < j.scan_index; } );
 
-      for (const auto& index : indices)
+      for (auto index : indices)
```

Comment:
can be const auto& I think

Reply:
Yes, since the `subdirs` String is moved and not the `index`.